templates/interface/filter-panels.interface.hbs

```diff
@@ -64,26 +65,75 @@
         </div>
       </div>
 
-      <div class="form-group clearfix" id="filter-value-type">
-        <div class="col-sm-4 control-label">
-          <label>Value type</label>
+      <div id="default-comparison">
```

Comment:
This template is repeated for each panel. The HTML template should not contain any IDs unless it's specific to each panel, e.g. `value-type-field-{{id}}`. There are several instances in this template. Please correct them all.